client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -16,34 +16,38 @@ import ZoneDetailsForm from '../../forms/zone-details-form';
 import { addZone } from '../../../state/zones/actions';
 import { settingsPath } from '../../../app/util';
 
-const ZoneCreator = ( {
-	saveZone,
-	siteId,
-	siteSlug,
-	translate,
-} ) => (
-	<div>
-		<HeaderCake backHref={ `${ settingsPath }/${ siteSlug }` }>
-			{ translate( 'Add a zone' ) }
-		</HeaderCake>
-
-		<ZoneDetailsForm label={ translate( 'New zone' ) } siteId={ siteId } onSubmit={ saveZone } />
-	</div>
-);
+class ZoneCreator extends PureComponent {
+
+	static propTypes = {
+		addZone: PropTypes.func.isRequired,
+		siteId: PropTypes.number,
+		siteSlug: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+	}
+
+	save = ( form, data ) => this.props.addZone( this.props.siteId, form, data );
+
+	render() {
+		const { siteId, siteSlug, translate } = this.props;
+
+		return (
+			<div>
+				<HeaderCake backHref={ `${ settingsPath }/${ siteSlug }` }>
+					{ translate( 'Add a zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Add a DNS Zone' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/zoninator-edit-zone)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).